src/main/java/consoleCommands/ConsoleCommands.java

```diff
@@ -0,0 +1,122 @@
+package consoleCommands;
+
+import exception.*;
+import jdk.dynalink.beans.StaticClass;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class ConsoleCommands {
+    public static final String COMMAND_MARKED = "mark";
+    public static final String COMMAND_TODO = "todo";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String COMMAND_EVENT = "event";
+    public static final String COMMAND_DEADLINE_BY = "/by";
+    public static final String COMMAND_EVENT_AT = "/at";
+    public static final String ADDED_MESSAGE = "Got it. I've added this task:";
+    public static final String DELETED_MESSAGE = "Noted. I've removed this task:";
+    public static final String MARKED_MESSAGE = "Nice! I've marked this task as done:";
+    public static final String UNMARKED_MESSAGE = "OK, I've marked this task as not done yet:";
+    public static final String HELLO_MESSAGE = "Hello! I'm Duke\n" + "What can I do for you?";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+    public static void markStatus(String input, ArrayList<Task> Array) throws TaskDoesNotExistException {
+        printLine();
+        String[] instructions = input.split(" ");
+        int index = Integer.parseInt(instructions[1]);
+        if (index > Array.size()) {
+            throw new TaskDoesNotExistException();
+        }
+        if (instructions[0].equals(COMMAND_MARKED)) {
+            Array.get(index-1).isDone = true;
+            System.out.println(MARKED_MESSAGE);
+        } else {
+            Array.get(index-1).isDone = false;
+            System.out.println(UNMARKED_MESSAGE);
+        }
+        System.out.println("  " + Array.get(index-1));
+        printLine();
+    }
+    public static void printList(ArrayList<Task> Array) {
```

Comment:
Parameter naming is off convention/standards.
If it is a constant it should be `ARRAY`, if not it should be `array`.
Also, a better name would be useful.